.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,3 +23,5 @@
 - [ ] My change requires a change to the documentation or CHANGELOG.
 - [ ] I have updated the documentation/CHANGELOG accordingly.
 - [ ] I have created a feature (non-master) branch for my PR.
+For code changes:
+- [ ] I have written tests for my changes.
```

Comment:
I would remove this 'header/label' personally. People can just remove the checkbox if it isn't applicable.
```suggestion
- [ ] I have written tests for my code changes.
```